cmd/iam.go

```diff
@@ -210,7 +210,7 @@ func (sys *IAMSys) Load(ctx context.Context) error {
 func (sys *IAMSys) Init(ctx context.Context, objAPI ObjectLayer, etcdClient *etcd.Client, iamRefreshInterval time.Duration) {
 	bootstrapTrace("IAM initialization started")
 	globalServerConfigMu.RLock()
-	s := globalServerConfig
+	s := globalServerConfig.Clone()
```

Comment:
Initialization never happens, again and again, it is only ever called once. So there was no need to clone()

But this is a harmless change